src/view/faq.njk

```diff
@@ -242,7 +242,7 @@
             <p>The Trustlines validator auction is a Dutch auction held to discover the fair opportunity cost value, i.e. price of a validator slot. The Ethereum addresses of successful bidders in a successful auction will be included in the validator set.</p>
 
             <p>
-              <strong>Acceptable tokens for a bid</strong>: The auction is a smart contract on the Ethereum mainchain. Note that the smart contract will not accept any other tokens than ETH as a bid.</p>
+              <strong>Acceptable tokens for a bid</strong>: The auction is a smart contract on the Ethereum mainchain. Note that the smart contract will not accept any other tokens than TLN as a bid.</p>
```

Comment:
Somewhere we need the warning. DO NOT SEND TLN directly to the token contract, it will be locked.